include/sealib/container.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SEALIB_CONTAINER_H_
+#define SEALIB_CONTAINER_H_
+#include "sealib/_types.h"
+
+namespace Sealib {
+/**
+ * An container interface to be used by several classes in this library.
+ * @param T the element type that this container holds
+ * @author Simon Heuser
+ */
+template <class T, class I = uint>
+class Container {
```

Comment:
The Container looks like a Collections of values, where each value has an unique index.
How about calling in `Sequence` instead of `Container`?